### PR TITLE
Add reducers and actions

### PR DESCRIPTION
**Pull request by Lornakaboro**
## In this activity I did the following : 

- Added Redux Toolkit to my project.
- Created a directory to hold all my Redux logic at /src/redux.
- Configured a Redux store in /src/redux/store.js.
- Defined a slice of state for books that:
- Includes an array of books with an initial state of an empty array.
- Includes a reducer that adds a book to the array.

### CODE REVIEW

File: src/redux/categories/categoriesSlice.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+});
+
+const categoriesSlice = (state = initialState, action) => {
+  if (action.type === CHECK_STATUS) {
+    return 'Under construction';
+  }
+  return state;
+};
+
+export default categoriesSlice;
```
Comment (OyePriscilla) [inline src/redux/categories/categoriesSlice.js]: - [x] Please kindly implement this using redux toolkit (RTK) according to the project requirement, it seems like you're using the old method. ℹ️ 

Untitled

File: src/redux/books/booksSlice.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      return [...state, action.payload];
+
+    case REMOVE_BOOK:
+      return state.filter((bookItem) => bookItem.id !== action.payload);
+    default:
+      return state;
+  }
+};
+
+export default bookSlice;
```
Comment (OyePriscilla) [inline src/redux/books/booksSlice.js]: - [x] - [ ] Please kindly implement this using redux toolkit (RTK) according to the project requirement, it seems like you're using the old method. ℹ️ 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Lornakaboro 👏 👏 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Use the ducks pattern for Redux files ✔️ 
Configure Redux store ✔️ 

## Required Changes ♻️

- [x] @Lornakaboro Please kindly fix the issues broken link under the Contributing section of your README file. 👍 This will make your repo more engaging. ℹ️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Lornakaboro,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉
### Highlights
- Professional readme file ✔️
- Good file structure ✔️
approval


# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**